lib/runner.ts

```diff
@@ -1,5 +1,6 @@
 import {EventEmitter} from 'events';
 import * as q from 'q';
+import {Session} from 'selenium-webdriver';
```

Comment:
nit: add `promise as wdpromise` to this and replace all the instances of `webdriver.promise` below with `wdpromise`